vng_api_common/exception_handling.py

```diff
@@ -30,6 +30,14 @@ def _translate_exceptions(exc):
 
 def get_validation_errors(validation_errors: dict):
     for field_name, error_list in validation_errors.items():
+        # nested validation for fields where many=True
+        if isinstance(error_list, list):
+            for i, nested_error_dict in enumerate(error_list):
+                if isinstance(nested_error_dict, dict):
+                    for err in get_validation_errors(nested_error_dict):
+                        err['name'] = f"{underscore_to_camel(field_name)}.{i}.{err['name']}"
+                        yield err
+                continue
```

Comment:
I can't wrap my head around why this `continue` is after the if-statement, and not the for-loop.

Reply:
Me neither, removing it makes the tests still pass too

Following the code flow it will then fall into the last for-loop.

De-indenting it so it comes after the for loop and skips the outer for-loop breaks the tests, so I'll remove it.